packages/api/portfolioDrafts/application/getApplicationStep.ts

```diff
@@ -51,3 +51,5 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
     return DATABASE_ERROR;
   }
 }
+
+export const handler = middy(baseHandler).use(cors(CORS_CONFIGURATION));
```

Comment:
Why don't we use the JSONErrorHandlerMiddleware for the GET?

Reply:
@kylelaker-ccpo I was under the impression that we didn't really need it from a previous discussion, so I did not included it. I will add it. I believe @zachclark-ccpo is working on a unified response with AT-6734 that might not include the `JSONErrorHandlerMiddleware`.

Reply:
Oh I'm not challenging the decision -- I'm just curious about the justification

Reply:
I'll leave it in to stay consistent with the others routes.